CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to @ftrack/api
+
+Thank you for wanting to contribute to our Javascript API!
+
+Please clone this repository, and follow these instructions to make sure you are in a clean spot before coding:
+
+Using node version as reflected in `.nvmrc`, run:
+
+- yarn install
+- yarn test
+
+If the tests passes, you should be good to go. Please submit a PR with your changes and we will look at them as soon as possible.
+
+## Publishing a release
+
+_This is only relevant for ftrack employees when preparing and publishing a new release._
+
+To publish a new release, follow these steps:
+
+Locally:
+
+- `git switch -c release/<new-release-tag>`
+- `yarn version <new-release-tag>`
+- `git push --tags`
```

Comment:
I believe you want `git push --follow-tags` (or `git push && git push --tags`) to push both the commit _and_ the tag.